programs/uxd/src/instructions/credix_lp/collect_profit_of_credix_lp_depository.rs

```diff
@@ -406,13 +406,13 @@ pub(crate) fn handler(ctx: Context<CollectProfitOfCredixLpDepository>) -> Result
         collateral_amount: collateral_amount_after_precision_loss,
     });
 
-    // Accouting for depository
+    // Accounting for depository
     let mut depository = ctx.accounts.depository.load_mut()?;
     depository.update_onchain_accounting_following_profit_collection(
```

Comment:
micro-nit: I noted that we used "profits" in most places, but in other places we use "profit", I created a PR to rename credix inconsistencies:
https://github.com/UXDProtocol/uxd-program/pull/223

Might be nice to name everything "profits" to match everywhere